src/reconstruct/reconstruction.cpp

```diff
@@ -49,6 +49,10 @@ Reconstruction::Reconstruction(MeshBlock *pmb, ParameterInput *pin)
   // Read and set type of spatial reconstruction
   // --------------------------------
   std::string input_recon = pin->GetOrAddString("parthenon/mesh", "xorder", "2");
+  // Avoid pmb indirection
+  const IndexDomain entire = IndexDomain::entire;
+  const IndexDomain interior = IndexDomain::interior;
+  const IndexShape cellbounds = pmb->cellbounds;
```

Comment:
Should this one be a reference? e.g.,
```suggestion
const IndexShape &cellbounds = pmb->cellbounds;
```